From eae95e80146c27ebadae92154d2421e1c57e6396 Mon Sep 17 00:00:00 2001
From: Tim Blechmann <tim.blechmann@qt.io>
Date: Thu, 25 Sep 2025 18:46:36 +0800
Subject: [PATCH 084/239] tests: silence some clang warnings

* -Wunused-private-field
* unused lambda capture

Change-Id: Ie9b75731530f3f999ac788c4b4dee4f1b12cae43
Reviewed-by: Ulf Hermann <ulf.hermann@qt.io>
(cherry picked from commit eee7404be0c9a5007bddfaaa1e1c3a627e76f4ce)
---
 .../auto/qml/qmltyperegistrar/inaccessible/invisible.h |  2 +-
 tests/auto/quick/scenegraph/tst_scenegraph.cpp         | 10 ++++++----
 2 files changed, 7 insertions(+), 5 deletions(-)

diff --git a/tests/auto/qml/qmltyperegistrar/inaccessible/invisible.h b/tests/auto/qml/qmltyperegistrar/inaccessible/invisible.h
index d1b96d7fc1..7be4b598fe 100644
--- a/tests/auto/qml/qmltyperegistrar/inaccessible/invisible.h
+++ b/tests/auto/qml/qmltyperegistrar/inaccessible/invisible.h
@@ -12,7 +12,7 @@ class InvisibleBase : public QObject
 public:
     InvisibleBase(QObject *parent = nullptr) : QObject(parent) {}
 private:
-    int m_a = 12;
+    Q_DECL_UNUSED_MEMBER int m_a = 12;
 };
 
 #endif // BASE_H
diff --git a/tests/auto/quick/scenegraph/tst_scenegraph.cpp b/tests/auto/quick/scenegraph/tst_scenegraph.cpp
index 18a0d33819..c26104260d 100644
--- a/tests/auto/quick/scenegraph/tst_scenegraph.cpp
+++ b/tests/auto/quick/scenegraph/tst_scenegraph.cpp
@@ -944,10 +944,12 @@ void tst_SceneGraph::unexposeDuringPolish()
     view.show();
     QVERIFY(QTest::qWaitForWindowExposed(&view));
 
-    connect(&view, &QQuickWindow::afterAnimating, this,
-            [this, &view]() { QVERIFY(view.isExposed()); });
-    connect(&view, &QQuickWindow::beforeSynchronizing, this,
-            [this, &view]() { QVERIFY(view.isExposed()); });
+    connect(&view, &QQuickWindow::afterAnimating, this, [&view]() {
+        QVERIFY(view.isExposed());
+    });
+    connect(&view, &QQuickWindow::beforeSynchronizing, this, [&view]() {
+        QVERIFY(view.isExposed());
+    });
 
     QQuickItem *rootItem = view.rootObject();
     rootItem->setProperty("contentsRect", QRectF());
-- 
2.51.2

